### PR TITLE
Migrate to maplibre

### PR DESCRIPTION
**Pull request by atomiomi**

### CODE REVIEW

File: packages/map/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
+## v5.0.0 Breaking changes
```
Comment (peterhass) [inline packages/map/CHANGELOG.md:1]: why is it a breaking change?

Comment (atomiomi) [reply packages/map/CHANGELOG.md:1]: You don't really need to change anything from user perspective but still, we are migrating from one library to another

Comment (peterhass) [reply packages/map/CHANGELOG.md:1]: If users don't need to do anything, then it's no breaking change

Comment (atomiomi) [reply packages/map/CHANGELOG.md:1]: Hm, I already published the package would be difficult to revert, let's have this version

Comment (peterhass) [reply packages/map/CHANGELOG.md:1]: 😭

Comment (suprMax) [reply packages/map/CHANGELOG.md:1]: no problem just bump the dependencies in the apps

Comment (peterhass) [reply packages/map/CHANGELOG.md:1]: At least add a comment here in CHANGELOG explaining that nothing needs to be changed

**Pull request merged**